config/lint.xml

```diff
@@ -43,4 +43,5 @@
 
     <issue id="ContentDescription" severity="ignore"/>
     <issue id="HardwareIds" severity="ignore"/>
+    <issue id="MissingPermission" severity="ignore"/>
```

Comment:
I think it's good to disable this check in the PR since it flags unrelated things that are not really issues but let's file an issue to either fix the problems it flags or explicitly suppress the check where permission checks actually exist but aren't exactly where the linter expects.